bokeh/models/tools.py

```diff
@@ -30,39 +30,29 @@
 #-----------------------------------------------------------------------------
 # Boilerplate
 #-----------------------------------------------------------------------------
-import logging
+import logging # isort:skip
 log = logging.getLogger(__name__)
 
 #-----------------------------------------------------------------------------
 # Imports
 #-----------------------------------------------------------------------------
 
-# Standard library imports
-
-# External imports
-
 # Bokeh imports
-from ..core.enums import (Anchor, Dimension, Dimensions, Location,
-                          TooltipFieldFormatter, TooltipAttachment)
+from ..core.enums import Anchor, Dimension, Dimensions, Location, TooltipAttachment, TooltipFieldFormatter
 from ..core.has_props import abstract
-from ..core.properties import (
-    Auto, Bool, Color, Date, Datetime, Dict, Either, Enum, Image, Int, Float,
-    Percent, Instance, List, Seq, String, Tuple
-)
+from ..core.properties import (Auto, Bool, Color, Date, Datetime, Dict, Either, Enum, Float,
+                               Image, Instance, Int, List, Percent, Seq, String, Tuple)
 from ..core.validation import error
-from ..core.validation.errors import (
-    INCOMPATIBLE_BOX_EDIT_RENDERER, INCOMPATIBLE_POINT_DRAW_RENDERER,
-    INCOMPATIBLE_POLY_DRAW_RENDERER, INCOMPATIBLE_POLY_EDIT_RENDERER,
-    INCOMPATIBLE_POLY_EDIT_VERTEX_RENDERER, NO_RANGE_TOOL_RANGES
-)
+from ..core.validation.errors import (INCOMPATIBLE_BOX_EDIT_RENDERER, INCOMPATIBLE_POINT_DRAW_RENDERER,
+                                      INCOMPATIBLE_POLY_DRAW_RENDERER, INCOMPATIBLE_POLY_EDIT_RENDERER,
+                                      INCOMPATIBLE_POLY_EDIT_VERTEX_RENDERER, NO_RANGE_TOOL_RANGES)
```

Comment:
I already expressed my disapproval of this style, yet it's being pushed further without proper discussion.

Reply:
This happens in a tiny fraction of cases, even when it does, it is not bad. I find this vastly, vastly easier to parse. When things hang under the lines it is very hard for me visually. I very rarely push/advocate on style but I am very firm on desiring to keep this sort default.